commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/display/context/CommerceOrderContentDisplayContext.java

```diff
@@ -349,6 +352,29 @@ public long getDisplayStyleGroupId() {
 		return _displayStyleGroupId;
 	}
 
+	public List<DropdownItem> getDropdownItems() {
+		List<DropdownItem> headerDropdownItems = new ArrayList<>();
+
+		DropdownItem headerDropdownItem = new DropdownItem();
```

Comment:
Should be named `DropdownItem headerDropdownItem1` -- see https://docs.google.com/document/d/188YFP0V4n9WJIPYZL3jGBRBvEsvBNmHOU97Fdd_g6Ew/edit#heading=h.urpkyx3jrv4y